lib/player.js

```diff
@@ -524,6 +522,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     /** @private {shaka.media.Playhead} */
     this.playhead_ = null;
 
+    /** @private {number} */
+    this.operationId_ = 0;
+
+    /** @private {!shaka.util.Mutex} */
+    this.mutex_ = new shaka.util.Mutex();
```

Comment:
Best practices for mutexes in any language is to document what the mutex protects.  That makes it easier to audit how/when they are acquired and what can be done while holding them.

Reply:
Added a string identifier.